charts/judge/values.yaml

```diff
@@ -0,0 +1,165 @@
+kratos:
+  enabled: true
+  kratos:
+    config:
+      selfservice:
+        allowed_return_urls:
+          - https://login.testifysec.localhost
+          - https://kratos.testifysec.localhost
+          - https://judge.testifysec.localhost
+          - https://localhost
+          - https://judge.testifysec.localhost:8077
+
+        methods:
+          oidc:
+            config:
+              providers:
+                - id: github
+                  provider: github
+                  client_id: your-github-client-id
+                  client_secret: your-github-client-secret
+                  issuer_url: https://github.com
+                  mapper_url: file:///etc/config/kratos/github.jsonnet
+                  scope:
+                    - user
+                  # -and/or 
+                - id: gitlab
+                  provider: gitlab
+                  client_id: your-gitlab-client-id
+                  client_secret: your-gitlab-client-secret
+                  issuer_url: https://gitlab.com
+                  mapper_url: file:///etc/config/kratos/gitlab.jsonnet
+                  scope:
+                    - openid
+                    - profile
+                    - email
+                    - read_user
+                    - read_api
+                    - read_repository
+archivista:
+  enabled: true
+
+kratos-selfservice-ui-node:
+  enabled: true
+
+judge-api:
+  enabled: true
+
+judge-web:
+  enabled: true
+
+dex:
+  enabled: true
+  ingress:
+    enabled: true
+    className: "nginx"
+    annotations:
+      cert-manager.io/cluster-issuer: tls-ca-issuer
+      kubernetes.io/ssl-redirect: "true"
+      nginx.ingress.kubernetes.io/rewrite-target: /
+    hosts:
+      - host: dex.testifysec.localhost
+        paths:
+          - path: /
+            pathType: Prefix
+    tls:
+      - secretName: dex-tls-secret
+        hosts:
+          - dex.testifysec.localhost
+  config:
+    issuer: https://dex.testifysec.localhost
+    storage:
+      type: memory
+    logger:
+      level: debug
+      format: json
+    oauth2:
+      responseTypes: [ "code" ]
+      skipApprovalScreen: true
+      alwaysShowLoginScreen: true
+    staticClients:
+      - id: testifysec
+        public: true
+        name: 'testifysec'
+    connectors:
+      - type: gitlab
+        id: gitlab-fulcio-test
+        name: Gitlab
+        config:
+          clientID: your-gitlab-client-id
+          clientSecret: your-gitlab-client-secret
+          redirectURI: https://dex.testifysec.localhost/callback
+
+fulcio:
```

Comment:
There's no fulcio chart in this pr

Reply:
correct, but do you think there's any harm in including it here?
If someone comes and uses our public charts and then decides to use fulcio in the umbrella, the values will already be provided as a form of self-documentation.